frontend/src/lib/components/LemonFileInput/LemonFileInput.scss

```diff
@@ -1,7 +1,15 @@
 .FileDropTarget {
-    border: 3px dashed transparent;
+    position: relative;
 }
 
-.FileDropTarget--active {
-    border-color: var(--primary);
+.FileDropTarget--active::after {
```

Comment:
so this is to avoid the risk of the content shifting as the border appears and disappears?

Reply:
Yeah, this way the border is completely independent, and the input can use all of modal's width. It's just this tiny thing that was bugging me that there was such a large padding, which isn't present in any other modal :D